.bandit

```diff
@@ -0,0 +1 @@
+skips: ['B101']
```

Comment:
I'm happy with this as we have little to gain from running using Python's [so-called] optimisation.